project_task_code/README.rst

```diff
@@ -1,19 +1,31 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+=========================
 Sequential code for tasks
 =========================
 
 * This module adds a sequential code for tasks.
 
+Usage
+=====
+
+To use this module, you need to:
```

Comment:
Add here that when you create a task in the menu X, you get a code saving it (you know, specifying points with `#.`).